modules/nextflow/src/main/groovy/nextflow/executor/FluxExecutor.groovy

```diff
@@ -61,9 +61,10 @@ class FluxExecutor extends AbstractGridExecutor {
     @Override
     List<String> getSubmitCommandLine(TaskRun task, Path scriptFile ) {
 
-        List<String> result = ['flux', 'mini', 'submit']
+        List<String> result = ['flux', 'submit']
         result << '--setattr=cwd=' + quote(task.workDir)
         result << '--job-name="' + getJobNameFor(task) + '"'
+        result << '--flags=waitable'
```

Comment:
This would require you to have `flux wait` somewhere, so I need more detail about why you are adding it here. The `flux submit` is the same as `flux mini submit` (I removed the mini early on, because it wasn't needed).

Reply:
Quoting original author:

---

`--flags=waitable` was not necessary (I thought that I had explained that already somewhere ... if not, I'm sorry). Background was that I didn't know whether nextflow itself is doing some blocking here anyhow ... Otherwise, with "flux sbatch" flux returns usually immediately to the shell prompt, and letting flux do its job in the background.

Initially, I feared that if I omit that waitable flag, nextflow would submit the task dependency tree of tasks to flux, and return. In that case, the slurm job had just finished, and the flux instance been killed by the slurm daemon. I was happy to learn that this fear was unsubstantiated.

---

So I think that we can remove this 👍🏻 

```suggestion
```